docs/guide/robot_sbc/setup_raspberry_pi.md

```diff
@@ -33,6 +33,7 @@ On Windows, with your memory card image burned and memory disc still inserted, y
 > Note: If __boot__ is not visible right away, try unplugging and re-inserting the memory card reader.
 
 * Start a text editor: `gedit` on Linux. Notepad++ on Windows. TextEdit on a Mac.
+* Possible `country` codes to use can be found [here](https://www.thinkpenguin.com/gnu-linux/country-code-list)
```

Comment:
It should shortly explain why it is needed (I believe it's mainly for wifi networks)